private-templates-service/client/src/index.css

```diff
@@ -1,5 +1,14 @@
+html {
+	padding-top: 56px;
```

Comment:
Guessing this won't be responsive - double check by resizing your browser window.

Reply:
I left it in because Khang is modifying the Navbar and I didn't want any conflicts. He will change this.